ocean/helium/data_transfer.py

```diff
@@ -159,6 +160,23 @@ def download_bytes(path, version=None):
     return body, meta
 
 
+def _calculate_etag(file_obj):
+    size = file_obj.stat().st_size
```

Comment:
can you throw a docstring here that describes the two etag formats for regular and multipart?